client/blocks/login/two-factor-authentication/security-key-form.jsx

```diff
@@ -31,8 +31,12 @@ class SecurityKeyForm extends Component {
 		isAuthenticating: false,
 	};
 
+	componentDidMount() {
+		setTimeout( () => this.initiateSecurityKeyAuthentication(), 100 );
```

Comment:
Why is the timeout necessary? 🤔 
Also, why 100ms?

Reply:
Removed the `setTimeout` statement in 1fb7185df4, I initially added it because I thought for some reason it was "needed".